File: Csharp/Baekjoon_History_Csharp/SourceCode/1259.cs
using System.Text;

namespace Fuc20
{
	public class Program
	{
		const string EOF = "0";
		public static void aMain()
		{
			string input = null;
			StringBuilder sb = new StringBuilder();

			int indexLeft = 0;
			int indexRight = 0;
			bool isPalindrome = true;

			while (true)
			{
				input = Console.ReadLine();

				if (input == EOF)
				{
					break;
				}

				indexLeft = 0;
				indexRight = input.Length - 1;
				isPalindrome = true;

				for (;indexLeft <= indexRight; indexLeft ++ , indexRight --)
				{
					if (input[indexLeft] != input[indexRight])
					{
						isPalindrome = false;
					}
				}

				sb.AppendLine(isPalindrome ? "yes" : "no");

			}

			Console.WriteLine(sb.ToString());



		}
	}
}

File: Cpp/Baekjoon_History_Cpp/SourceCode/1259.cpp
#include <stdio.h>
#include <array>


int main()
{

	char input[6];

	int indexLeft = 0;
	int indexRight = 0;

	bool isPalindrome = true;

	while (true)
	{
		

		scanf_s("%s" , &input , sizeof(input));
		if (input[0] == '0')
		{
			break;
		}
		indexLeft = 0;
		
		indexRight = 

		isPalindrome = true;

		for (; indexLeft <= indexRight; indexLeft++ , indexRight --)
		{
			if (input[indexLeft] != input[indexRight])
			{
				isPalindrome = false;
			}
		}
		
		puts(isPalindrome ? "yes" : "no");


	}
	


	return 0;
}